website/api/zkvm/quickstart.md

```diff
@@ -5,96 +5,74 @@ slug: ./quickstart
 ```bash
-cargo risczero new my_project
+cargo risczero new my_project --guest-name guest_code_for_zk_proof
 ```
 
-See the [cargo risczero] docs for other options for initialization.
+To run the zkVM on your own machine, we recommend at least 16 GB of RAM.
 
-## 3. Edit the boilerplate to build your first project
+## 3. Project structure
 
-In the codebase for your new project, you'll find a handful of places marked `TODO`, where you'll need to make some changes in order to build your first project.
+The [Hello World tutorial] contains step-by-step instructions on how to:
 
-Don't worry -- each `TODO` comes with instructions for what you need to do, and the [Hello World tutorial] contains step-by-step instructions.
+- [Share private data] between the host & guest.
+- [Share the guest's results] with the host publicly.
+- [Generate a receipt] and read its journal contents.
 
 ## 4. Quick Development: Leveraging Dev Mode
 
-During the development of your project, you might find that running your code can take a long time due to the proof generation process. To address this issue and allow for faster iterations of your code, we suggest utilizing [dev-mode]. This mode bypasses the time-consuming proof generation process. To activate dev-mode, simply set the environment variable `RISC0_DEV_MODE=true` when executing your project (other acceptable values include `1` and `yes`).
-
-Example:
+During the development of your project, running your code can take a long time due to the proof generation process. To address this issue and allow for faster iterations of your code, we suggest utilizing [Dev Mode]. This mode bypasses the time-consuming proof generation process. To activate dev-mode, set the environment variable `RISC0_DEV_MODE=true` when executing your project:
 
 ```bash
 RISC0_DEV_MODE=true cargo run --release
 ```
 
-For a deeper understanding of dev-mode and its safe usage, please refer to our page explaining [what is dev-mode]. Please note that dev-mode is only meant to be used during development and testing. It should **never** be used in production.
-
 ## 5. Real Proof Generation
 
-Once you've reached a point where you're ready to generate real proofs, you can do so by unseting the `RISC0_DEV_MODE` environment variable (or setting it to `RISC0_DEV_MODE=false`). We recommend that you additionally specify the feature flag `disable-dev-mode`, which will ensure that dev-mode is not accidentally enabled. Please consult more information about `disable-dev-mode` in the [feature flags] table, and the [dev-mode] page for more information.
-
-Proceeding with the example above, generating proofs locally would be achieved by running:
+Once you've reached a point where you're ready to generate real proofs, you can do so by setting the `RISC0_DEV_MODE=false`. We recommend that you specify the feature flag `disable-dev-mode`, which will ensure that dev-mode is not accidentally enabled. Generating proofs locally would be achieved by running the following:
```

Comment:
```suggestion
Once you've reached a point where you're ready to generate real proofs, you can do so by setting `RISC0_DEV_MODE=false`. We recommend that you specify the feature flag `disable-dev-mode`, which will ensure that dev-mode is not accidentally enabled. Generating proofs locally would be achieved by running the following:
```

Reply:
I don't now that I'd recommend using `disable-dev-mode`. But perhaps we could mention that it's a feature available if someone needs it.